### PR TITLE
freatue-1 development is done

### PR DESCRIPTION
**Pull request by giriediga**
you need to write as clear as possible about what features have you done here

### CODE REVIEW

File: README.MD [hunk 1]
```diff
@@ -1,2 +1,6 @@
 #### GIT
-THis is the first commit from main branch
\ No newline at end of file
+THis is the first commit from main branch
+
```
Comment (Subash02in) [inline README.MD:3]: is thissss line really required?

**Review by Subash02in — COMMENTED** [review]
is thiss really required.?

File: README.MD (hunk 1 above)
Comment (giriediga) [reply README.MD:3]: yes that is required

**Review by Subash02in — APPROVED** [review]
fine go ahead

**Pull request merged**